.github/workflows/release.yml

```diff
@@ -59,3 +61,36 @@ jobs:
           OVSX_PAT: ${{ secrets.OVSX_TOKEN }}
         run: |
           yarn ovsx:publish --pat $OVSX_PAT ./metals-${{ steps.get_version.outputs.NEW_VERSION }}.vsix
+
+  pre-release:
+    if: |
+      always() &&
+      github.event_name == 'push' && 
+      !contains(github.event.head_commit.message, 'scalameta/dependabot/') &&
+      !startsWith(github.event.head_commit.message, 'Bump')
```

Comment:
Could we isntead check if the author is scalameta bot or dependabot?

Reply:
Seems like `github.event_name == 'push' &&`, depending on what we want to do after release, is or isn't needed. See your other comment

Reply:
It is possible, but I'm not sure what that check should look like. Can I add a logging step, which will log all necessary information needed for this check? After the first PR's from that bots workflow can be tweaked